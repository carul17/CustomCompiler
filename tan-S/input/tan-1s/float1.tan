main {
    const r := 4.5 / 0.5;
    const a := 4.5 / 1.0;
    const b := 6.0 / 2.0;
    const s := 45 / 5;
    const z := 45 / 1;
    const y := 6 / 3;
	print r \n;
	print a \n;
	print b \n;
	print s \n;
	print z \n;
	print y \n;
}