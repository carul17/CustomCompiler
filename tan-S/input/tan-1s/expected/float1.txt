9.000000
4.500000
3.000000
9
45
2
